package.json

```
@@ -39,15 +42,15 @@
   ],
   "scripts": {
     "build": "npm run --workspaces build",
-    "check-license": "./scripts/check-license.sh",
+    "check-license": "node scripts/check-license.js",
     "coverage": "npm run --workspaces coverage",
-    "depcheck": "./scripts/run-depcheck.sh",
-    "eslint": "eslint --cache 'scripts/*.{js,jsx,ts,tsx}' 'packages/*/src/**/*.{js,jsx,ts,tsx}' 'packages/*/*.{js,jsx,ts,tsx,mts}'",
+    "depcheck": "node scripts/run-depcheck.js",
+    "eslint": "eslint --cache \"scripts/*.{js,jsx,ts,tsx}\" \"packages/*/src/**/*.{js,jsx,ts,tsx}\" \"packages/*/*.{js,jsx,ts,tsx,mts}\"",
     "fmt": "npm run prettier",
     "lint": "npm-run-all -ln --parallel prettier-lint tsc-lint eslint check-license",
     "prepare": "npm run --workspace @jaegertracing/plexus prepublishOnly",
-    "prettier": "prettier --write '{.,scripts}/*.{js,jsx,json,md,ts,tsx,mts}' 'packages/*/{src,demo/src}/**/!(layout.worker.bundled|react-vis).{css,js,jsx,json,md,ts,tsx}' 'packages/*/*.{css,js,jsx,json,md,ts,tsx,mts}'",
-    "prettier-lint": "prettier --list-different '{.,scripts}/*.{js,jsx,json,md,ts,tsx,mts}' 'packages/*/{src,demo/src}/**/!(layout.worker.bundled|react-vis).{css,js,jsx,json,md,ts,tsx}' 'packages/*/*.{css,js,jsx,json,md,ts,tsx,mts}' || (echo '*** PLEASE RUN npm run prettier AND RESUBMIT ***' && false)",
+    "prettier": "prettier --write \"**/*.{js,jsx,json,md,ts,tsx,mts,css}\" \"!packages/*/src/LayoutManager/layout.worker.bundled.js\" \"!**/node_modules/**\" \"!**/dist/**\" \"!**/lib/**\" \"!**/build/**\"",
```

```suggestion
    "prettier": "prettier --write \"**/*.{js,jsx,json,ts,tsx,mts,css}\" \"!packages/*/src/LayoutManager/layout.worker.bundled.js\" \"!**/node_modules/**\" \"!**/dist/**\" \"!**/lib/**\" \"!**/build/**\"",
```
why is this still touching YAML files?

---

it touched YAML file because i ran `npx prettier --write . ` this command touch every files

---

So in this case, running `npx prettier --write . `will format all supported files by default, including YAML files.
If you don’t want Prettier to touch YAML files, you should run the command defined in package.json
`npm run prettier`

---

in the main branch when I run `npm run prettier` no files are being changed (the current state of the repo already reflects prettier rules).

---

Please run npx prettier--write . In main repo

---

no it doesn't
```
$ npm run prettier

> jaeger-ui-monorepo@0.0.1 prettier
> prettier --write '{.,scripts}/*.{js,jsx,json,md,ts,tsx,mts}' 'packages/*/{src,demo/src}/**/!(layout.worker.bundled|react-vis).{css,js,jsx,json,md,ts,tsx}' 'packages/*/*.{css,js,jsx,json,md,ts,tsx,mts}'
 ...

$ gst
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
```

---

> i think we have to run npx prettier --write .

no we don't, that's why we have a command for it in package.json that configures prettier how we want it.